pkg/operator/controller/clustercompliancereport.go

```diff
@@ -0,0 +1,88 @@
+package controller
```

Comment:
I know that most of our controllers are in the `pkg/operator/controller` package, but I'd move it to the `pkg/complaince` to keep our code modular. Basically all functionality, whether it's utility or controller that's related to compliance reports is scoped to the `pkg/compliance` package.